app/models/container_image.rb

```diff
@@ -46,7 +46,7 @@ class ContainerImage < ApplicationRecord
   serialize :exposed_ports, Hash
   serialize :environment_variables, Hash
 
-  virtual_column :display_registry, :type => :string
+  virtual_delegate :display_registry, :to => "container_image_registry.full_name", :allow_nil => true, :type => :string
```

Comment:
What happens when this returns nil?  Is there a presentation-side change where it will say "Unknown image source", or does it leave it blank?